client-rpc/src/program.rs

```diff
@@ -93,18 +83,13 @@ pub fn find_string(args: &[String], target: &str) -> Option<usize> {
 #[allow(dead_code)]
 pub fn run_electron() {
     env_logger::init();
-    // "~/Electron", ".", "--network-id", "ab", "--network-type", "test"]
+    // "~/Electron", ".", "--chain-id", "ab", "--network-type", "test"]
     let args: Vec<String> = env::args().collect();
     log::info!("args={:?}", args);
     let mut options = Options::from_iter(vec![""].iter());
-    if let Some(a) = find_string(&args, "--network-id") {
-        options.network_id = args[a + 1].clone()
+    if let Some(a) = find_string(&args, "--chain-id") {
+        options.chain_id = args[a + 1].clone()
```

Comment:
@leejw51crypto I have changed this part which related to Electron, will it affect it?

Reply:
chain_id = test...ab
network_id=  ab
network_type = test

Reply:
so is  chain_id full id?,  or just changing name?

Reply:
ok

Reply:
chain_id is the full id, for reference

Reply:
i'll modify electron as another pull request

Reply:
yes